CONTRIBUTING.md

```diff
@@ -22,6 +22,41 @@ $ rubocop -V
 
 * Include any relevant code to the issue summary.
 
+## Creating Cops
+
+### Create the cop files
+
+1) Create your cop in `./lib/rubocop/cop/{type}/{name}.rb`
+2) Require your cop in `./lib/rubocop.rb`
+3) Your test should be created at `./spec/rubocop/cop/{type}/{name}_spec.rb`
+4) Add your rule to either the `config/disabled.yml` or `config/enabled.yml` file depending on if it should be enabled by default or not. Cops that are not based on style guide rules should be disabled (normally) by default.
+
+### Finding your offenses
+
+#### Investigate
+
+The [investigate method](lib/rubocop/cop/style/semicolon.rb#L11) gets passed a [processed source](lib/rubocop/processed_source.rb). This object can be asked questions about the source, retrieve all of the source tokens and get specific lines as strings.
```

Comment:
If `lib/rubocop/cop/style/semicolon.rb` changes in the future, the link will become irrelevant.
One way to solve this is to [bind to the current SHA](http://andrew.yurisich.com/work/2014/07/16/dont-link-that-line-number/).